src/components/ResourceForm/UserFormPage.tsx

```diff
@@ -120,9 +137,8 @@ const UserFormPage = ({
           accessLevel: role,
         },
       });
-      enqueueSnackbar('Team member edited.');
     }
-    history.replace('/manage/members');
+    history.push('/manage/members');
```

Comment:
I noticed that you changed a lot of the `history.replace` methods to `history.push`. Was this an intended change? It's only a subtle difference, so not sure if we intended to do this

Reply:
Yeah I changed it on purpose to push a new entry onto the history stack instead of replacing the current entry on the history stack which will prevent the user from accessing the previous URL when they click the browser back button